src/commands/autocomplete/index.ts

```diff
@@ -22,54 +22,38 @@ export default class Autocomplete extends AutocompleteBase {
     const shell = this.argv[0] || this.config.shell
     if (!shell) {
       cli.error('Error: Missing required argument shell')
+    } else if (shell !== 'bash' && shell !== 'zsh') {
+      cli.error(`Currently ${shell} is not a supported shell for autocomplete`)
     }
 
-    if (!this.flags['skip-instructions']) {
-      if (shell !== 'bash' && shell !== 'zsh') {
-        cli.error(`Currently ${shell} is not a supported shell for autocomplete`)
-      }
+    cli.action.start(`${cli.color.bold('Building the autocomplete cache')}`)
```

Comment:
`'Building the autocomplete cache...<spinner.'` @crcastle feel free to suggest different verbiage